.github/workflows/windows-end-to-end-tests.yml

```diff
@@ -37,7 +37,7 @@ jobs:
         run: pnpm i
 
       - name: Build CLI tool
-        run: pnpm build
+        run: pnpm build:packages
```

Comment:
So it seem as if Xero does not build on Windows, so using a packages only build script.